Foundation/NSString.swift

```diff
@@ -20,6 +20,16 @@ extension unichar : ExpressibleByUnicodeScalarLiteral {
     }
 }
 
+// Placeholder for a future implementation
```

Comment:
Is this comment correct? Haven't you provided the implementation?

Reply:
I’ll remove it from the rebase.

Reply:
The comment from the SDK implementation reads:

/// Returns a localized string, using the main bundle if one is not specified.

Maybe we can use that instead?

Reply:
Sure, added to the rebase